src/config.ts

```diff
@@ -51,15 +51,15 @@ export default {
     },
     server: {
       // support various ENV that uses PORT vs SERVER_PORT
-      port: env.PORT || env.SERVER_PORT,
+      port: env.get('PORT').asInt() || env.get('SERVER_PORT').asInt(),
       // allow Access-Control-Allow-Origin
       corsOptions: {
         maxAge: 86400,
         origin: whitelist
       }
     },
     logging: {
-      level: env.LOGGING_LEVEL
+      level: env.get('LOGGING_LEVEL').required().asString()
     },
     //
     // If you want to set up conformance statement with security enabled
```

Comment:
These comments are probably outdated because the following code is uncommented.

Reply:
Not sure what a conformance statement is so I'm a little hesitant to delete this comment

Reply:
Just @ing you @plarocque4 since the comment was last added/touched in https://github.com/mcode/REMS/commit/1b1c7a2bc064070bb82bf672bc3af0281c53942e; I wasn't sure if the comment is still relevant?